tests/TestEntry.cpp

```diff
@@ -126,6 +126,24 @@ void TestEntry::testClone()
     QCOMPARE(entryCloneHistory->timeInfo().creationTime(), entryOrg->timeInfo().creationTime());
     delete entryCloneHistory;
 
+    Entry* entryCloneUserRef = entryOrg->clone(Entry::CloneUserAsRef);
```

Comment:
QScopedPointer

Reply:
All the `Entry` variables declared in this file need to be refactored as well, like the comment above: Should I do it or should we ship it like this and fix in the next version?

Reply:
Just do it. It's basically search and replace.